src/EnergyPlus/MixedAir.cc

```diff
@@ -1670,14 +1670,17 @@ namespace MixedAir {
 				for ( groupNum = 1; groupNum <= NumGroups; ++groupNum ) {
 					VentMechZoneName( groupNum ) = AlphArray( ( groupNum - 1 ) * 3 + 5 );
 
-					for ( OutAirNum = 1; OutAirNum <= NumOAControllers; ++OutAirNum ) {
-						if ( OAController( OutAirNum ).VentilationMechanicalName == VentilationMechanical( VentMechNum ).Name && VentilationMechanical( VentMechNum ).DCVFlag ) {
-							AirLoopControlInfo( OutAirNum ).AirLoopDCVFlag = true;
-						} else {
-							AirLoopControlInfo( OutAirNum ).AirLoopDCVFlag = false;
+					if ( AirLoopControlInfo.size() >= NumOAControllers ){
+						for ( OutAirNum = 1; OutAirNum <= NumOAControllers; ++OutAirNum ) {
+							if ( OAController( OutAirNum ).VentilationMechanicalName == VentilationMechanical( VentMechNum ).Name && VentilationMechanical( VentMechNum ).DCVFlag ) {
+								AirLoopControlInfo( OutAirNum ).AirLoopDCVFlag = true;
```

Comment:
This appears to assume that the index of the AirLoopControlInfo array is the same as the OAController array.  But will that be true if there is an OA controller in an ERV?